infra/tf/configs/metrics.yml

```diff
@@ -0,0 +1,25 @@
+#cloud-config
+groups:
+  - ubuntu: [root,sys]
+
+runcmd:
+  - cd /root
+  - ufw allow 80
+  - ufw allow 443
+  - echo "IPINFO_TOKEN=${ipinfo_token}" >> /etc/environment
+  - source /etc/environment
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
```

Comment:
most of the docker scripts here can be replaced with 

```
curl -fsSL https://get.docker.com -o get-docker.sh
sh get-docker.sh
```

Reply:
https://github.com/fleek-network/ursa/pull/81/commits/e73d912b63a13d5be09c7146e7140440fdb25814